HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -175,7 +175,7 @@ copymethod:: Collection -minIndex
 
 
 method::find
-If the strong::sublist:: exists in the receiver (in the specified order), at an offset greater than or equal to the initial strong::offset::, then return the starting index.
+If the strong::sublist:: exists in the receiver (in the specified order), at an offset greater than or equal to the initial strong::offset::, then return the starting index. The type of the sublist needs to match the type of the list to search in. Elements are checked for equality (not for identity).
```

Comment:
Better not talk of "types", but of "class".

Reply:
Thanks for pointing that out. I did not know about the different meaning of type and class. 
"Same kind" sounds very generic to me though. I would propose  "The sublist must be of the same kind (class) as the list to search in."

Reply:
yes, I agree, that's good.